README.md

```diff
@@ -9,11 +9,19 @@ Explore the tree and most of the modifications you may want to make should be fa
 
 Check out this repository:
 
-`git clone git@github.com:TwilightCoders/iocage-plugin-fail2ban.git`
+`git clone https://github.com/TwilightCoders/iocage-plugin-fail2ban.git`
```

Comment:
I'd like to encourage `ssh` over `https` as a standard amongst these projects.

Reply:
The problem is that one needs to have an ssh key setup for Github to access the ssh url.
Only https works for anonymous...

https://stackoverflow.com/a/46993922/869402

Reply:
You know, that's a really good point! I never considered that, and I appreciate the perspective. I'm going to modify the other `iocage` plugins I'm creating for the same reason. Thanks!